arangod/Pregel/Worker/State.h

```diff
@@ -64,6 +68,7 @@ struct WorkerState {
         resultActor(std::move(resultActor)),
         statusActor(std::move(statusActor)),
         metricsActor(std::move(metricsActor)) {
+    executionState = std::make_unique<Initial<V, E, M>>(self, *this);
     config->updateConfig(specifications);
 
     if (messageCombiner) {
```

Comment:
this whole if statement could possibly be done in the constructor of the Computing state.